GameServerLib/GameObjects/AttackableUnits/AI/ObjAIBase.cs

```diff
@@ -51,6 +51,7 @@ public class ObjAiBase : AttackableUnit, IObjAiBase
         public bool IsCastingSpell { get; set; }
         public bool IsMelee { get; set; }
         public bool IsDashing { get; protected set; }
+        public float DashSpeed { get; set; }
```

Comment:
I think we can start with at least protected set and see if "public" is really needed.